pdf/core/utils.go

```diff
@@ -38,6 +38,18 @@ func getUniDocVersion() string {
 	return common.Version
 }
 
+// GetObjectNums returns the object numbers of the PDF objects in the file
+func (this *PdfParser) GetObjectNums() (map[int]int, error) {
```

Comment:
What is the purpose of getting all object numbers (outside of this package)?

Why use a map?  Would it be simpler to just return an []int with all the numbers ?

A related functionality is the Inspect() function.  Would it be better to expose a single function that iterates through all the objects and calls a user-supplied function?  That could accommodate both use cases (more generic).